code/nodes/featureCreation/fresh/featureCreate.q

```diff
@@ -8,7 +8,7 @@
 // @return {tab} features created in accordance with the FRESH feature creation procedure.
 featureCreation.fresh.create:{[feat;cfg]
   aggCols:cfg`aggcols;
-  params:get cfg`funcs;
+  params:utils.qpyFuncSearch cfg`funcs;
```

Comment:
This can also be a `get` call, the return of this call is a keyed table outlining all the functions to be applied, I think it's unlikely that a user will define this python side for use